README.md

```diff
@@ -193,6 +193,13 @@ Print QR Code for public address.
 
 When enabled, listhen tries to listen to all network interfaces. You can also enable this option using `--host` CLI flag.
 
+### `ipc`
+
+- Default: `listhen`
```

Comment:
I'm a bit unsure about the doc update. The default is actually `undefined` not `listhen`; if activated with `--ipc`, then it's `listhen`.

If the description is confusing, it cloud be improved a bit